host/xtest/xtest_10000.c

```diff
@@ -280,7 +280,7 @@ static const uint8_t pbkdf2_3_dkm[] = {
 #define pbkdf2_4_password pbkdf2_1_password
 #define pbkdf2_4_salt pbkdf2_1_salt
 #define pbkdf2_4_iteration_count 16777216
-static const uint8_t pbkdf2_4_dkm[] = {
+static const uint8_t pbkdf2_4_dkm[] __maybe_unused = {
```

Comment:
Builds, yes, absolutely, both on gcc5.x and [gcc6.x](https://travis-ci.org/jbech-linaro/build). But, now when you're saying it, I'm also a bit puzzled where it comes from. I took for granted that we had that ready to be used. When I compile the file and look at the pre-processor output (`-E -dM`). I see
```c
...
#define __maybe_unused __attribute__((unused)) 
...
```
It's used in [other places](https://github.com/OP-TEE/optee_test/search?utf8=%E2%9C%93&q=__maybe_unused) to under xtest. But no define and I cannot find anything about it when grepping in the toolchain folder. Weird! Maybe we should define it in optee_test somewhere? `xtest_test.h`? Or some new common h-file shared with both the TA's and the host.